### ISSUE TITLE
Classic temple - Pray and sacrifice

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1249]
In DK1 the temple was used both to make creatures pray to get happy and to sacrifice creatures. In DK2 the casino was introduced for the happiness part and the temple was used to get more mana and to sacrifice creatures.
OpenDungeons has renamed the dungeon heart as the temple, and does not have the classic temple room at all.

However, as you want to have a multiplayer focused game, I want to propose to introduce the classic temple. Either DK1 or DK2 style, but in case of the former the casino can be removed.

In the DK games you mainly got creatures as a result for temple sacrifices, but with FX(the fan expansion) more effects were added (e.g. sacrifice a fly and a hound to get a temporary speed effect on all your creatures). I would propose this is the main direction to take this in, with a crucial addition:
- There are both positive and negative sacrifice outcomes
- The outcomes do not effect the player who sacrifices the creature but the **owner of the creatures** sacrificed.

This allows for many strategic options in a multiplayer game, you can give your own creatures a temporary buff and try to force an engagement, but it also becomes an interesting alternative when you manage to capture enemy creatures and you'll have a choice to either convert OR sacrifice these creatures to get a debuff on your rivals creatures.

**Comment by hwoarangmy** [comment]
I agree the temple was not used enough in the original DK games and its behaviour should be improved. However, we have no fitting model AFAIK so let's hope someone would upload something fitting on open game art.
